Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,247 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
```

Comment:
Be careful what you commit and add to a PR - this PR should only contain the week2 homework changes, not week1.